bucket/connect.json

```
@@ -0,0 +1,9 @@
+{
+    "version": "1.100",
+    "description": "connect --- simple relaying command via proxy.",
+    "homepage": "http://www.meadowy.org/~gotoh/projects/connect",
```

Since homepage is not available anymore I would suggest to use archive web as homepage.

```suggestion
    "homepage": "https://web.archive.org/web/20080516100455/http://www.meadowy.org/~gotoh/projects/connect",
```